IAMOTZ/More-recipes
# #152082320 API for posting recipe

#### What does this PR do?

It should enable the server to post recipe to a dummy database.

#### Description of Task to be completed?
User can add a recipe to his/her catalog(API endpoint).
#### How should this be manually tested?

- clone the repo
- cd into the server directory
- run the following commands
- `npm install` to install dependencies
- `npm run start:dev` to start the server on port 3000
- On postman or any API testing tool, send post request to `/api/recipes`
- The post request should be accompanied by a body containing: 
`{ id: "", title: "", body: ""}`
- Fix in whatever you like in the id, title and body properties
#### Any background context you want to provide?
None
#### What are the relevant pivotal tracker stories?
#152082663
#### Screenshots (if appropriate)
None
#### Questions:
None
